data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1366,23 +1411,26 @@ mission "Hai Reveal [B13] Freedom"
 				goto finished
 			label knowhai
 			`	They incline their head respectfully. "The Hai are perhaps the greatest valuers of truth among us. If they have set you to this, then they will be honored by your diligence.`
+				goto finished
 			label knowsyndi
 			`	They focus on you closely. "The Syndicate is perhaps the most impressive machine ever wrought by man. It is worthy of both respect and fear, and will do to people as any machine does to the product of its design. Treat it as such and it will always do precisely what you expect it to; both good and bad.`
+				goto finished
 			label knowduty
 			`	They lift their head and smile. "A promise made is a promise kept. If your word is your bond then it would not do for me to stand in its way.`
+				goto finished
 			label finished
 			`	"The missing leader you seek is embedded within the Syndicate and is known by the name Terry Adrianopoulos."`
 			choice
-				`	(Be surprised.)`
+				`	"Terry did this!?"`
 					goto surprised
-				`	(You suspected.)`
+				`	"I suspected as much."`
 			`	Their words scratch an itch of suspicion that you didn't realize had been irritating you until now. They smirk knowingly. "I have merely confirmed that which you already guessed. Very well, I need say no more. This matter is yours to do with as you will." They leave you to ponder your course.`
 				goto done
 			label surprised
-			`	Your face must have revealed your response, for they smile at you with subtle mirth. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
+			`	They smile at you with subtle mirth at your surprise. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
```

Comment:
```suggestion
			`	They smile with subtle mirth at your surprise. "I see you know of this person but did not suspect. I think it is to you, and you alone, I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
```

This version carries a bit more accusatory weight towards Terry. It's a good place to have a subtle line as to this speaker's views of her, but I'm not sure which side of that line we want them on.